bundle/Form/Payload/InformationCollectionStruct.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Netgen\Bundle\InformationCollectionBundle\Form\Payload;
+
+final class InformationCollectionStruct
+{
+    /**
+     * @var mixed[] An array of field values like[$fieldDefIdentifier]
+     */
+    private array $collectedData = [];
+
+    /**
+     * Returns value for $fieldDefIdentifier.
+     *
+     * @return mixed
+     */
+    public function getCollectedFieldValue(string $fieldDefIdentifier)
+    {
+        return $this->collectedData[$fieldDefIdentifier] ?? null;
+    }
+
+    /**
+     * This method returns the complete fields collection.
+     */
+    public function getCollectedFields(): array
+    {
+        return $this->collectedData;
+    }
+
+    /**
+     * Sets value for $fieldDefIdentifier.
+     *
+     * @param mixed $value
+     */
+    public function setCollectedFieldValue(string $fieldDefIdentifier, $value): void
```

Comment:
Add type hint for `$value` and remove the hint from docblock.

Please got through all added code and fix stuff like this.

Reply:
solved for this specific class cc986d9e